extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
These groups shouldn't be separated from each other, only `java`, `javax` and the static imports should be in a separate group.